sections/acknowledgements.include

```diff
@@ -485,106 +317,139 @@
   David Kendal,
   David MacDonald,
   David Matja,
+  David Newton,
   David Remahl,
+  David Singer,
   David Smith,
   David Storey,
   David Vest,
   David Woolley,
-  DeWitt Clinton,
   Dean Edridge,
   Dean Edwards,
+  "dbol55",  <!-- https://github.com/dbol55-->
   Debi Orton,
+  Deborah Kaplan,
+  Denis Ah-Kang,
   Derek Featherstone,
+  Derek Koziol,
   Devarshi Pant,
   Devdatta,
+  DeWitt Clinton,
+  Diego Lopes Lima,
   Dimitri Glazkov,
   Dimitry Golubovsky,
+  Dirkjan Ochtman,
   Dirk Pranke,
   Dirk Schulze,
-  Dirkjan Ochtman,
   Divya Manian,
+  Dmitry Shuralyov,
   Dmitry Titov,
   dolphinling,
+  Domenic Denicola,
   Dominic Mazzoni,
   Dominique Hazaël-Massieux,
+  Dom Talbot,
   Don Brutzman,
+  Don Hollander,
   Doron Rosenberg,
+  Doug Brunelle,
   Doug Kramer,
   Doug Simpkinson,
+  Drew McLellan,
   Drew Wilson,
+  "duckware", <!-- https://github.com/duckware-->
   Dylan Barrell,
   Edmund Lai,
-  Eduard Pascual,
   Eduardo Vela,
+  Eduard Pascual,
   Edward O'Connor,
```

Comment:
You can drop this line.

Reply:
yup. (Sorry, thought that had been done already)